crates/wasi-common/src/old/snapshot_0/hostcalls_impl/misc.rs

```diff
@@ -342,10 +342,14 @@ pub(crate) struct FdEventData<'a> {
     pub(crate) userdata: wasi::__wasi_userdata_t,
 }
 
+// Not all implementations use this implementation.
+#[allow(dead_code)]
```

Comment:
Should this perhaps just be deleted?

Reply:
There's no dep on this code in Lucet, so if Wasmtime doesn't intend to use it, lets delete it.

Reply:
Done.